### PR TITLE
Added readme for UI library

### PR DESCRIPTION
**Pull request by Palaksharma23**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,94 @@
-# ui-library
-Elegant design system &amp; Vue.js 3 UI library for moja global
+<div align="center">
+<img src="https://moja.global/wp-content/uploads/2021/03/Asset-66@4x.png" alt="Moja Global UI Library logo" height ="auto" width="200" /> 
+
+<br />
+
+<h1 align="center">UI Library</h1>
+
+ [npm](https://www.npmjs.com/package/@moja-global/mojaglobal-ui)   [yarn](https://yarnpkg.com/package/@moja-global/mojaglobal-ui)
```
Comment (HarshCasper) [inline README.md:8]: Use proper badges for this purpose: https://badge.fury.io/

File: README.md [hunk 2]
```diff
@@ -1,2 +1,94 @@
...
+ </div>
+
+## About The Project
```
Comment (HarshCasper) [inline README.md:11]:
```suggestion
## About
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,94 @@
...
+
+It is an Elegant design system built on Vue.js version 3 for moja global. It is an open-source UI library having pre-written code for various widely used components across all the repositories of moja global that developers and users can use in the projects just by installing the package and running import commands
+ 
+## Built with
```
Comment (HarshCasper) [inline README.md:15]: Remove this

File: README.md [hunk 4]
```diff
@@ -1,2 +1,94 @@
...
+- Datepicker
+- Dropdown
+- Footer
+- Modal
+- Navbar
+- Slider 
+- Sponsor
+- Toggle
+
+Detailed description about each of the component and how to use the component can be found  [here](https://ui-library-au5vxcs10-moja-global.vercel.app/?path=/story/example-accordion--primary)
```
Comment (HarshCasper) [inline README.md:34]: Put this as a table instead showing all the components that are implemented and planned.

Comment (HarshCasper) [inline README.md:34]: Put this like: `Find detailed description on our [Storybook setup](link)`

File: mojaglobal-ui/README.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
+<div align="center">
```
Comment (HarshCasper) [inline mojaglobal-ui/README.md:1]: Is this file needed?

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: mojaglobal-ui/README.md (hunk 5 above)
Comment (Palaksharma23) [reply mojaglobal-ui/README.md:1]: Yes sir, this file is needed
The README.md file inside the mojaglobal-ui directory will appear on the npm package 
The README.md file in the root project will appear on the github repository

**Review by legendarykamal — APPROVED** [review]
LGTM :+1: good to merge.

**Comment by HarshCasper** [comment]
I have used your docs to make a new PR here: https://github.com/moja-global/ui-library/pull/50

Since you had disabled the maintainers to make edits it was not possible for me to make any changes and reviewing the PR would have delayed it further. I hope its fine.
